Article/Article.md

```diff
@@ -230,3 +148,85 @@ The second check has been the introduction of a different gender equality indica
 - Lastly, the most interesting result is that the gender differences residualised on the economic develooment don't show a dependency on the GDI (also residualised on economic development), while the vice-versa applies (meaning, there is a strong and statistically significant correlation between gender differences and economic development, both residualised on the GDI). This result arises many questions that are beyond the scope of this analysis.
 
 # References
+
+
+
+# Suppl.
```

Comment:
We need to check if the supplementary material can be given in this way or in another file. I think they want it in a separate file (would also make sense to me at least)